pkg/cmd/helm/release/release.go

```diff
@@ -216,7 +216,7 @@ func (o *Options) Validate() error {
 	if o.GitClient == nil {
 		o.GitClient = cli.NewCLIClient("", o.CommandRunner)
 	}
-	requirements, err := variablefinders.FindRequirements(o.GitClient, o.JXClient, o.Namespace, o.Dir, "", "")
+	requirements, err := variablefinders.FindRequirements(o.GitClient, o.JXClient, o.Namespace, o.Dir, "", "", "", "")
```

Comment:
If we always want to set `cloneType = "full"`, then this would fail. I can explicitly set it here if we don't want `cloneType` to ever be undefined.

Changing `jx gitops helm release` is out-of-scope for these changes, so we can set and forget?